src/printer/relay_text_printer.cc

```diff
@@ -330,6 +330,14 @@ Doc RelayTextPrinter::VisitExpr_(const ConstantNode* op) {
       return ScalarLiteral(dtype, static_cast<const uint8_t*>(op->data->data)[0]);
     }
   }
+  const PackedFunc* fprint = runtime::Registry::Get("relay._constant_text");
```

Comment:
Is `relay._constant_text ` defined by users?

Reply:
I think it is a global function that third party can plug in and customize the printing for constants

Reply:
Yeah, I am okay with this. I think another possible way could use a user-defined annotator and pass it to the pretty printer. One example could be something like:

```python
def constant_annotator():
    def _annotate(e):
        if isinstance(e, relay.Constant):
            # Print the constant
        else:
            return ""
    return _annotate

# Given an IRModule m or an expression m
print(m.astext(show_meta_data=False, annotate=constant_annotator()))
```